src/alert-prelude.c

```diff
@@ -618,6 +661,20 @@ static TmEcode AlertPreludeThreadInit(ThreadVars *t, void *initdata, void **data
 
     /** Use the Ouput Context */
     aun->ctx = ((OutputCtx *)initdata)->data;
+    
+    /** Create a per-thread idmef analyzer */
+    if (idmef_analyzer_new(&aun->analyzer) < 0) {
+        SCLogError(SC_ERR_INITIALIZATION,
+                   "Error creating idmef analyzer for Prelude.");
+        SCReturnInt(TM_ECODE_FAILED);
```

Comment:
it's not a huge deal but I don't like adding output to my QA. Cppcheck says: [src/alert-prelude.c:676]: (error) Memory leak: aun